contracts/pair/src/contract.rs

```diff
@@ -211,6 +219,7 @@ impl LiquidityPoolTrait for LiquidityPool {
                     min_b,
                     pool_balance_a,
                     pool_balance_b,
+                    Decimal::bps(custom_slippage_bps.unwrap_or(1)),
```

Comment:
https://www.investopedia.com/terms/b/basispoint.asp
1 BPS is not a 1%.

Reply:
that explains it greatly, thanks!